src/components/DataTable/Cells/users.js

```diff
@@ -19,13 +22,13 @@ const email = {
 const location = {
   id: 'location',
   Header: 'Location',
-  accessor: user => user.countryDescription,
+  accessor: user => countryText[user.countryId],
```

Comment:
I'm not sure about this approach, using array index positions to match up with the Ids.